openreview/conference/templates/programchairWebfield.js

```diff
@@ -1226,8 +1274,14 @@ var displayPaperStatusTable = function() {
 
     var filteredRows;
     if (searchText) {
-      filteredRows = _.orderBy(_.filter(rowData, filterFunc), sortOptions['Paper_Number'], 'asc');
-      matchingNoteIds = filteredRows.map(function(row) { return row.note.id; });
+      if(isQueryMode){
+        ({ filteredRows, queryIsInvalid } = Webfield.filterCollections(rowData, searchText.slice(1), filterOperators, propertiesAllowed, 'note.id'))
```

Comment:
thanks. 
besides the newly added code, there's also a saveAs call at the end which seems to have an extra comma and would fail with syntax error in ie.

so it looks like for the past 10 months no pc had used internet explorer to access the pc console.
maybe it's time to revisit support of internet explorer